mne/decoding/tests/test_time_gen.py

```diff
@@ -171,6 +171,25 @@ def test_generalization_across_time():
     assert_true(len(gat.test_times_['slices'][0]) == 15 ==
                 np.shape(gat.scores_)[1])
 
+    # Test score_mode
+    gat.score_mode = 'foo'
+    assert_raises(ValueError, gat.score, epochs)
+    gat.score_mode = 'per-fold'
+    scores = gat.score(epochs)
+    assert_array_equal(np.shape(scores), [15, 15, 5])
+    gat.score_mode = 'no-cv'
```

Comment:
mode-wise seems a good way of calling referring to the modes

On Fri, Apr 29, 2016 at 4:34 PM Eric Larson <notifications@github.com>
wrote:

> In mne/decoding/tests/test_time_gen.py
> <https://github.com/mne-tools/mne-python/pull/3143#discussion_r61586953>:
>
> > @@ -171,6 +171,25 @@ def test_generalization_across_time():
> >      assert_true(len(gat.test_times_['slices'][0]) == 15 ==
> >                  np.shape(gat.scores_)[1])
> >
> > +    # Test score_mode
> > +    gat.score_mode = 'foo'
> > +    assert_raises(ValueError, gat.score, epochs)
> > +    gat.score_mode = 'per-fold'
> > +    scores = gat.score(epochs)
> > +    assert_array_equal(np.shape(scores), [15, 15, 5])
> > +    gat.score_mode = 'no-cv'
>
> Seems okay to me as long as the docstring is clear
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3143/files/2c0fa419e56dd3ab6a5fe679b573acda56d21bf0#r61586953>
>